spec/features/observer_spec.rb

```diff
@@ -3,7 +3,7 @@
     expect(CommunicartMailer).to receive_message_chain(:on_observer_added, :deliver_later)
 
     work_order = create(:ncr_work_order)
-    observer = create(:user)
+    observer = create(:user, client_slug: 'ncr')
```

Comment:
I kinda like the explicit attribute-setting, so i don't have to guess what other magic something like `:ncr` might be applying.